goal/index.html

```diff
@@ -0,0 +1,154 @@
+            name="title"
+            placeholder="eg: Testing setup for dashboard"
+            id="title"
+            autofocus
+          />
+        </div>
+        <div class="inputBox">
+          <label for="task-name" class="editable"
+            >Task Name:<span class="required">*</span></label
+          >
+          <input
+            required
+            class="notEditing"
+            type="text"
+            name="task-name"
+            placeholder="eg: Testing setup for dashboard"
+            id="task-name"
+            autofocus
+          />
+        </div>
+        <div class="inputBox">
+          <label for="links" class="editable">Links:</label>
+          <input
+            type="text"
+            name="links"
+            id="links"
+            placeholder="Comma separated links"
+            class="notEditing"
+            autofocus
+          />
+        </div>
+        <div class="inputBox">
+          <label for="user-role" class="editable"
+            >User Role:<span class="required">*</span></label
+          >
+          <select
+            required
+            class="input notEditing"
+            name="user-role"
+            id="user-role"
+          >
+            <option disabled selected>Select a category</option>
+            <option value="MEMBER">Member</option>
+            <option value="ADMIN">Admin</option>
+            <option value="DEVELOPER">Developer</option>
+            <option value="TESTER">Tester</option>
+            <option value="NEWCOMER">Newcomer</option>
+            <option value="DESIGNER">Designer</option>
+          </select>
+        </div>
+        <div class="inputBox">
+          <label for="username" class="editable"
+            >Username:<span class="required">*</span></label
+          >
+          <input
+            required
+            class="input notEditing"
+            type="text"
+            name="username"
+            placeholder="eg: Shravan"
+            id="username"
+            autofocus
+          />
+        </div>
+        <div class="inputBox">
+          <label for="priority" class="editable"
+            >Priority:<span class="required">*</span></label
+          >
+          <select
+            required
+            class="input notEditing"
+            id="priority"
+            name="priority"
+          >
+            <option disabled selected>Select the priority</option>
+            <option value="HIGH">High</option>
+            <option value="MEDIUM">Medium</option>
+            <option value="LOW">Low</option>
+          </select>
+        </div>
+        <div class="inputBox">
+          <label for="type" class="editable"
+            >Type:<span class="required">*</span></label
+          >
+          <select required class="input notEditing" id="type" name="type">
+            <option disabled selected>Select a type</option>
+            <option value="FEATURE">Feature</option>
+            <option value="BUG">Bug</option>
```

Comment:
adding `id` to tags which will share the same styling (here priority, type, user-role) is not a good idea. Use class instead. That way you'll have just one class name that you can add to all 3 of them

Reply:
changed id to class in some elements